smart-contracts/.env.example

```diff
@@ -11,7 +11,7 @@ LOCAL_PROVIDER="http://localhost:7545"
 # Local validator count must be at least 1 and no more than 8 
 LOCAL_VALIDATOR_COUNT = 3
 # The initial power associated with each local validator
-LOCAL_INITIAL_VALIDATOR_POWERS = 50,1,1
+LOCAL_INITIAL_VALIDATOR_POWERS = 33,33,33
 
```

Comment:
some test case assume the first validator has the most of power, then the claim or peophecy can be finilized after the transaction issued from fist validator.

Reply:
This is only a `.env.example` and should be made as real world as possible and not used for tests. You will see in the `smart-contracts/migrations/2_next.js` script I have set it up with default if these values arent set and these will then kick in for any test run.